lib/RGBot.js

```diff
@@ -14,7 +14,6 @@ const { Vec3 } = require('vec3');
  *    <i>Mineflayer API documentation - https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md </i><br>
```

Comment:
Won't let me comment on the line, but on line 10, there is a `ß` character that we should remove

Reply:
I'm actually impressed, I have no idea how to even type that character... Removed